code/modules/surgery/organs/lungs.dm

```diff
@@ -336,10 +337,9 @@
 		if(healium_pp > gas_stimulation_min)
 			if(prob(15))
 				to_chat(H, "<span class='alert'>Your head starts spinning and your lungs burn!</span>")
-				SEND_SIGNAL(owner, COMSIG_ADD_MOOD_EVENT, "chemical_euphoria", /datum/mood_event/chemical_euphoria)
+				healium_euphoria = EUPHORIA_ACTIVE
 				H.emote("gasp")
-		else
-			SEND_SIGNAL(owner, COMSIG_CLEAR_MOOD_EVENT, "chemical_euphoria")
+
```

Comment:
You uh, you forgot something here I think

Reply:
huh?

Reply:
nevermind the huh, i'm stupid

Reply:
should be fine now no?

Reply:
:)